snapshots/sample/app/src/main/kotlin/com/emergetools/snapshots/sample/ui/TextRowWithIcon.kt

```diff
@@ -25,8 +25,7 @@ fun TextRowWithIcon(
 }
 
 @Preview
-@FontScalePreviews
-@LocalePreviews
+@SnapshotTestingPreview
```

Comment:
One nit: Could we make this a separate preview to make sure we can test both the outputs of stacked multipreviews directly on the composable and from the stacked multipreview annotation?